content/en/getting_started/dashboards/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Getting Started with Dashboards
+kind: documentation
+further_reading:
+- link: "https://learn.datadoghq.com/enrol/index.php?id=8"
+  tag: "Learning"
+  text: "Self-paced online learning: Building Better Dashboards"
+- link: "/dashboards/"
+  tag: "Documentation"
+  text: "Dashboards basics"
+- link: "/notebooks/"
+  tag: "Documentation"
+  text: "Tell a story about data with Notebooks"
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kinds of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
+
+Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those conversations into the same dashboard. Create separate dashboards to show answers to different kinds of questions.
+
+This guide gets you started on a path to making dashboards. These basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
+
+## Prerequisites
+
+If you haven't already, create a [Datadog account][1]. Install an agent on a host, and an integration for something running on that host.
```

Comment:
```suggestion
If you haven't already, create a [Datadog account][1]. Install the Agent on a host, and an integration for something running on that host.
```

Reply:
Maybe some links to the agent setup / integrations page here, just to make it easier for people to get up and running.